docs/ide-setup.md

```diff
@@ -47,4 +41,29 @@ Add the Argo schema setting `yaml.schemas`:
 
 That's it. Open an Argo YAML file and you should see smarter behaviour, including type errors and context-sensitive autocomplete.
 
-![VScode Example Functionality](assets/vscode-ide-step-4-example-functionality.png)
\ No newline at end of file
+![VScode Example Functionality](assets/vscode-ide-step-4-example-functionality.png)
+
+## Validating Argo YAML using CRD Validation in IntelliJ Ultimate Edition
```

Comment:
would it be possible to replace these with the JSON schema instructions for IntelliJ?